gammapy/analysis/config.py

```diff
@@ -154,7 +170,6 @@ class WcsConfig(GammapyBaseConfig):
 
 class GeomConfig(GammapyBaseConfig):
     wcs: WcsConfig = WcsConfig()
-    selection: SelectionConfig = SelectionConfig()
```

Comment:
See comment above on keeping this parameter in the settings for cutouts and not use the one for the safe mask.

Reply:
Normally the `offset_max` used by `cutout` here should be directly connected to the `SafeMaskMaker`. The parameter config is probably more logically found on the `SafeMaskConfig`.